src/components/BlogForm.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { addNewEntry, updateEntry } from '../redux/actions/blogActions';
 import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
```

Comment:
Why?

Reply:
i forgot to delete it after i used hoc for route protection. it's all good now